### DISCUSSION TITLE
Migration to organization

### DISCUSSION THREAD

**Discussion by Neustradamus** [rustdesk/rustdesk#1830] [General]
It is possible to migrate the personal account to organization?

Converting:
- https://help.github.com/articles/converting-a-user-into-an-organization/

It is possible to rename this account and create the organization and move/transfer the repository to the organization.

Rename account:
- https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username

Create an organization:
- https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch

Transfer a repository:
- https://help.github.com/en/github/administering-a-repository/transferring-a-repository

Like:
- http://github.com/gnome
- http://github.com/kde
- ...

**Comment by Neustradamus** [comment]
@rustdesk: Have you progressed on it?

**Comment by Neustradamus** [comment]
@rustdesk: Have you progressed on it?

**Comment by Neustradamus** [comment]
@rustdesk: Have you progressed on it?

**Comment by Croydon** [comment]
This would help with being more transparent about who is actually committing, pushing and discussing on GitHub.
Transparency is a must when it comes to building up trust.
